test/operator/test_operator.py

```diff
@@ -86,6 +86,20 @@ def _loc(*args):
     ["XX", "YZ", "IZ"], [0.1, 0.2, -1.4]
 )
 
+hi = nk.hilbert.Fermions2nd(5)
+operators["FermionOperator2nd"] = nk.operator.FermionOperator2nd(
+    hi,
+    terms=(((0, 1), (3, 0)), ((3, 1), (0, 0))),
```

Comment:
I wonder if it would be easy to make use of a more direct syntax, say using c(0)*c_dag(3) rather than ((0,0),(3,1)) etc ?

Reply:
Yes, I'd suggest allowing you to also pass strings:
"0^ 3" in your example, where ^ represents a dagger.

Reply:
I made a suggestion in recent commit (with example in the docstring)